nifi-extension-bundles/nifi-standard-services/nifi-oauth2-provider-bundle/pom.xml

```diff
@@ -27,4 +27,17 @@
         <module>nifi-oauth2-provider-service</module>
         <module>nifi-oauth2-provider-nar</module>
     </modules>
+    <repositories>
+        <!-- Shibboleth Repository required for OpenSAML -->
```

Comment:
The Controller Service does not use OpenSAML, so is this required? It seems like it could be removed with the removal of `nifi-web-security`.